### DISCUSSION TITLE
Limit Video Storage/Delete Old Videos

### DISCUSSION THREAD

**Discussion by DennisFaucher** [Motion-Project/motion#1555] [Q&A] [answered]
Hi, is there a built-in method to only keep N days/MB of videos or do we need to write our own cron job to remove old videos? TIA.

**Comment by Mr-Dave** [comment]
There is no such built-in method

**Comment by DennisFaucher** [comment, marked as answer]
Thanks. I wrote a cron job to delete any files older than today.

**Reply by PanderMusubi** [reply]
Would you like to share the cron line?

**Reply by DennisFaucher** [reply]
I have since deleted that VM, but I had created a shell script to be run daily that looked something like this:
find /path/to/files* -mtime +5 -exec rm {} \;
That deletes files older than 5 days
